Assets/MRTK/Providers/LeapMotion/Editor/ConfigurationChecker/LeapMotionConfigurationChecker.cs

```diff
@@ -82,11 +82,13 @@ private static bool ReconcileLeapMotionDefine()
             if (files.Length > 0)
             {
                 ScriptUtilities.AppendScriptingDefinitions(BuildTargetGroup.Standalone, Definitions);
+                ScriptUtilities.AppendScriptingDefinitions(BuildTargetGroup.WSA, Definitions);
                 return true;
             }
             else
             {
                 ScriptUtilities.RemoveScriptingDefinitions(BuildTargetGroup.Standalone, Definitions);
+                ScriptUtilities.RemoveScriptingDefinitions(BuildTargetGroup.WSA, Definitions);
                 return false;
             }
         }
```

Comment:
I do not think we should add WSA to the configuration checker for only in editor usage. A UWP build is not supported for leap so if we kept WSA there, we would need to add guards to prevent a UWP build.   With the PR in it's current state, a UWP build completes, when it should not.

Reply:
@CDiaz-MS Does the provider's assembly only building for editor and Windows Standalone not prevent it from being included in a UWP build? It doesn't look like it's the scripting define that excludes the DLL, or even prevents the code from running in a UWP build:

https://github.com/microsoft/MixedRealityToolkit-Unity/blob/9e60cb3f7850f61bb91a0dc2b5016e42ca54930f/Assets/MRTK/Providers/LeapMotion/Microsoft.MixedReality.Toolkit.Providers.LeapMotion.asmdef#L6-L10

Reply:
So, it sounds like the UWP build completing is expected, since the Leap code isn't included in the build regardless of the `#if` define.

Reply:
Guards were indeed not needed => removed.
I also changed the example to display a message that Leapmotion is only supported in editor when in a UWP build.